lib/features/notes/presentation/pages/note_read_only_page.dart

```diff
@@ -110,7 +117,13 @@ class _NotesReadOnlyPageState extends State<NotesReadOnlyPage> {
           automaticallyImplyLeading: false,
           actions: const [
             NoteSaveButton(),
-            ToggleReadWriteButton(pageName: PageName.NoteReadOnlyPage)
+            ToggleReadWriteButton(pageName: PageName.NoteReadOnlyPage),
+            IconButton(
+              icon: Icon(Icons.play_arrow), // Play button icon
+              onPressed: () {
+                _speak(notesBloc.state.title! + ". " + notesBloc.state.controller!.document.toPlainText());
+              },
+            ),
```

Comment:
- Extract this into a new widget `ReadNotes` in `lib/features/notes/presentation/widgets`
- The icon should change to `pause` icon, once the audio starts playing
- When clicked on `pause` icon the audio should be paused / stopped. It looks like pausing is bit tricky on Android, so you can use `flutterTts.stop()`

Reply:
Placement of this widget should be after `NoteSaveButton` as shown in screenshot. We want to keep `ToggleReadWriteButton` to the rightmost so that its easy to toggle between read and write modes